azure-pipelines.yaml

```diff
@@ -91,6 +91,37 @@ parameters:
       - plan
       - apply
 
+  # Used for building MI infrastructure across the environments
+  - name: environment_components
+    type: object
+    default:
+      - env: "sbox"
+        serviceConnection: "DCD-CFTAPPS-SBOX"
```

Comment:
For depends on, do we want them all to be sequential, they shouldnt rely on each other in any way?
If yes then sbox needs to depend on the `core infrastructure` stage so that the MIs are always after the core deployment.

Reply:
Ah that's a better idea,I was thinking all depends on a successful sbox run - but can have sbox run depend on stage before to make that the case

Reply:
I don't believe they're sequential this way , more just sbox > all nonprod > prod so we don't do something silly in all envs

![image](https://github.com/hmcts/version-reporter-services/assets/47995122/7dcb2403-828b-4212-996b-3d949d2d2f49)